### ISSUE TITLE
ReceivedHeader no value in By

### ISSUE THREAD

**Issue by mariuszkrzaczkowski** [zbateson/mail-mime-parser#162]
```
Received: from xcv.gurbuzsrc.com ([69.81.93.69])
	by mail.yetiforce.com with esmtp (Exim 4.94)
	(envelope-from <xxcv@gurbuzsrc.com>)
	id 1kfCyx-0002Zp-BY
	for vbc@yetiforce.com; Wed, 18 Nov 2020 03:14:03 +0100
```

```
FromName: xcv.gurbuzsrc.com
FromHostname: 
FromAddress: 69.81.93.69
ByName: 
ByHostname: 
ByAddress: 
with: esmtp
getComments: Exim 4.94 | envelope-from 
```

how to fix it ?

**Comment by zbateson** [comment]
Hmmm, from what I can tell from my comments 'getByName' should return mail.yetiforce.com, and at first glance this looks like a bug.  I'll have to investigate further, thanks for reporting.

**Comment by zbateson** [comment]
Sorry for the delay, that's actually Not sure if something was fixed since the 2.0 release, but this seems to be working now (added a test to confirm)
